safe_transaction_service/history/tests/test_internal_tx_indexer.py

```diff
@@ -304,16 +305,15 @@ def test_tx_processor_using_internal_tx_indexer_with_existing_safe(self):
         tx_processor.process_decoded_transactions(
             InternalTxDecoded.objects.pending_for_safes()
         )
-        safe_contract: SafeContract = SafeContract.objects.first()
-        self.assertGreater(safe_contract.erc20_block_number, 0)
-        safe_contract.erc20_block_number = 0
-        safe_contract.save(update_fields=["erc20_block_number"])
+        self.assertEqual(
+            IndexingStatus.objects.get_erc20_721_indexing_status().block_number, 0
+        )
+        IndexingStatus.objects.set_erc20_721_indexing_status(0)
```

Comment:
`IndexingStatus.objects.get_erc20_721_indexing_status().block_number` if indexingStatus is not created then is writting the `min(master_copy_block_number)` and if this last value doesn't exist is writting 0, then why are you setting 0 once again?